t/Makefile.am

```diff
@@ -128,6 +128,11 @@ TESTS += \
 	python/t0006-request.py \
 	python/t0007-watchers.py \
 	python/t0008-jsc.py
+
+if HAVE_FLUX_SECURITY
+TESTS += python/t0009-security.py
```

Comment:
Also add to `check_SCRIPTS` ?

Reply:
Good catch!  That hopefully solves the distcheck failure.